lib/fileutils.js

```diff
@@ -43,3 +68,29 @@ function which(exe, cb){
     cb(!!exePath)
   })
 }
+
+exports.findableByWhichOrModule = findableByWhichOrModule
+function findableByWhichOrModule(cb){
+  var browser = this
+
+  findableByWhich.call(browser, function(findable){
```

Comment:
I think the local version should be preferred over the global version. So overwriting works on a per project basis.